cmd/root.go

```diff
@@ -335,7 +325,28 @@ func mergeEnvs(envs []fn.Env, envToUpdate *util.OrderedMap, envToRemove []string
 
 // setPathFlag ensures common text/wording when the --path flag is used
 func setPathFlag(cmd *cobra.Command) {
-	cmd.Flags().StringP("path", "p", cwd(), "Path to the project directory (Env: $FUNC_PATH)")
+	cmd.Flags().StringP("path", "p", ".", "Path to the project directory (Env: $FUNC_PATH)")
+}
+
+// getPathFlag returns the value of the --path flag.
+// The special value '.' is returned as the abolute path to the current
+// working directory.
+func getPathFlag() string {
```

Comment:
Maybe we don't need this function at all.
The `.` path is already considered to be `cwd`.
I see some value in pinning absolute path if `cwd` is changed during execution but `func` is not changing `cwd`.

Reply:
Only tests are changing `cwd`.

Reply:
> Maybe we don't need this function at all.

I see your point, but do like using `os.Getwd()` to resolve it

Reply:
Yes, but we may avoid challenges in the future if the core executes without dependency on the parent process being in given working directory.  By converting the `.` to an absolute path in the CLI, we are at least treating it that way